scripts/generate-google3-docs/resource-reference/overview.md

```diff
@@ -374,6 +374,10 @@ issues for {{product_name_short}}.
   <td><a href="/container-analysis/docs/">{{container_analysis_name}}</a></td>
   <td><a href="/config-connector/docs/reference/resource-docs/containeranalysis/containeranalysisnote.md">ContainerAnalysisNote</a></td>
 </tr>
+<tr>
```

Comment:
Thank you for moving the entry down, @gemmahou ! From the overview page (https://cloud.google.com/config-connector/docs/reference/overview) it looks like we follow the alphabetical order of the product, so this one should probably be moved to the line right before GKE Hub?

Reply:
Seems like we order the resources by its Google cloud product name first then KCC resource name. Updated!